liberty-archetype-mp/src/test/resources/projects/it/archetype.properties

```diff
@@ -0,0 +1,10 @@
+groupId=test.wlp
+artifactId=ServletSample
+version=0.1-SNAPSHOT
+libertyGroupId=io.openliberty
+libertyArtifactId=openliberty-runtime
+wlpRuntimeVersion=17.0.0.4
+wlpPluginVersion=2.1.2
+package=${groupId}
```

Comment:
What do you do to see the folder with that name?  I'm not seeing that problem.  Also, all the other archetypes do this.

Reply:
In the Travis log you can see this output when the project gets built: 

```
Project created from Archetype in dir: /home/travis/build/WASdev/ci.maven/liberty-archetype-mp/target/test-classes/projects/it/project/ServletSample
```

I built the project locally and when I go into the `ServletSample` folder I see the folder with name `${groupId}`.

I haven't worked on any of the other archetypes before, but I can take a look at them to see if this is also happening.

Reply:
Yes, please could you try one of the other archetypes and see if you encounter the same problem.  Because I don't see this, I'm not sure how to go about reproducing/fixing it.  Thanks.